frame/transaction-payment/src/lib.rs

```diff
@@ -734,6 +744,7 @@ where
 			T::OnChargeTransaction::correct_and_deposit_fee(
 				&who, info, post_info, actual_fee, tip, imbalance,
 			)?;
+			Pallet::<T>::deposit_event(Event::<T>::TransactionPaid { who, fee: actual_fee, tip });
```

Comment:
`actual_fee` includes the tip here, so I would make it explicit in the event/documentation.
https://github.com/paritytech/substrate/blob/ef661427da2f9cec68f871ddf8e2de32bc0402ac/frame/transaction-payment/src/types.rs#L81

Reply:
I.e. the inclusion fee is `actual_fee - tip`